infrastructure/api-mgmt.tf

```diff
@@ -30,8 +30,8 @@ module "api_mgmt" {
   api_mgmt_rg   = "core-infra-${var.env}"
   revision      = "1"
   product_id    = "${module.api_mgmt_product.product_id}"
-  display_name  = "Get SAS token for storage account access"
-  path          = "token"
+  display_name  = "Blob Router API"
+  path          = "reform-scan"
```

Comment:
is path change intentional ? Third party supplier will need to update ? This is anyways to retrieve SAS token for should be fine as it is ?

Reply:
yes, intentional. the construct is \<api-gw\>/\<path\>/\<swagger\> so `app-gw-url/reform-scan/token/{service_name}`

Reply:
Consumer(supplier) would have an impact is it agreed change ?

Reply:
nobody agreed on anything in the first place. all api gateways require path as it's same url. bulk scan processor has `bulk-scan`, fees - `fees-api` and so on. reform scan product (this component is under that flag) has `reform-scan` after this pr

Reply:
ok I think there is no choice we will have to update the path